test/AutoDiff/leakchecking.swift

```diff
@@ -8,6 +8,13 @@ import DifferentiationUnittest
 
 var LeakCheckingTests = TestSuite("LeakChecking")
 
+/// Execute body, check expect leak count, and reset global leak count.
```

Comment:
```suggestion
/// Execute body, check expected leak count, and reset global leak count.
```
Maybe fix in follow-up PR to unblock progress.

Reply:
Fixed in https://github.com/apple/swift/pull/25057/commits/ca36c09d80d1db06495e3d68feea790c8db9ac96.